imitation/trainer.py

```diff
@@ -39,12 +38,11 @@ def __init__(self,
                env: Union[gym.Env, str],
                gen_policy: BaseRLModel,
                discrim: DiscrimNet,
-               expert_policies: Sequence[BaseRLModel],
+               expert_rollouts: Tuple[np.ndarray, np.ndarray, np.ndarray],
```

Comment:
Consider a named type, e.g. `Rollout = Tuple[np.ndarray, np.ndarray, np.ndarray]`, somewhere else in the file?

Reply:
I will leave as is, especially since some Rollout things expect obs-act-obs-rew, and others expect obs-act-obs.